### PR TITLE
Extend roadmap: input file=type

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ This scenario loads a web page, enters text into a form field via the keyboard,
 - [x] Everything from the previous scenario
 - [x] [Emulating keyboard input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
 - [x] [Emulating mouse input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
+- [ ] [Filling out input type=file](https://github.com/w3c/webdriver-bidi/issues/494)
```
Comment (mathiasbynens) [inline roadmap.md:45]:
```suggestion
- [ ] [Filling out `<input type=file>`](https://github.com/w3c/webdriver-bidi/issues/494)
```

**Review by mathiasbynens — APPROVED** [review]

File: roadmap.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ This scenario loads a web page, enters text into a form field via the keyboard,
 - [x] Everything from the previous scenario
 - [x] [Emulating keyboard input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
 - [x] [Emulating mouse input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
+- [ ] [Filling out `<input type=file>`](https://github.com/w3c/webdriver-bidi/issues/494)
```
Comment (jgraham) [inline roadmap.md:45]: "Filling out" doesn't exactly convey the scope to me. It seems like the proposed scope is "Set local file paths on `<input type=file>`" i.e. this doesn't cover uploading file data or similar.

Comment (OrKoN) [inline roadmap.md:45]:
```suggestion
- [ ] [Ability to test and fill out file inputs](https://github.com/w3c/webdriver-bidi/issues/494)
```

Comment (OrKoN) [reply roadmap.md:45]: @jgraham WDYT?

Comment (whimboo) [reply roadmap.md:45]: There is now another issue for that with a proposal. As such we can close this one.

File: roadmap.md [hunk 3]
```diff
@@ -42,6 +42,7 @@ This scenario loads a web page, enters text into a form field via the keyboard,
 - [x] Everything from the previous scenario
 - [x] [Emulating keyboard input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
 - [x] [Emulating mouse input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
+- [ ] [Ability to test and fill out file inputs](https://github.com/w3c/webdriver-bidi/issues/494)
```
Comment (OrKoN) [inline roadmap.md:45]:
```suggestion
- [x] [Ability to set files on file inputs](https://w3c.github.io/webdriver-bidi/#command-input-setFiles)
```

**Review by whimboo — APPROVED** [review]

**Pull request merged**